scene/main/scene_tree.h

```diff
@@ -214,9 +214,9 @@ class SceneTree : public MainLoop {
 	void remove_from_group(const StringName &p_group, Node *p_node);
 	void make_group_changed(const StringName &p_group);
 
-	void _process_group(ProcessGroup *p_group, bool p_physics);
-	void _process_groups_thread(uint32_t p_index, bool p_physics);
-	void _process(bool p_physics);
+	void _process_group(ProcessGroup *p_group, bool p_physics, bool p_internal);
+	void _process_groups_thread(uint32_t p_index, Pair<bool, bool> p_physics_internal);
```

Comment:
Out of curiosity, what's the rationale for using a `Pair<bool, bool>` here as opposed to passing two bool arguments like for the other two `_process*`?

Reply:
`WorkerThreadPool::add_template_group_task()` allows for a single user data argument. At first, I made a custom `ProcessingPass` struct and then I thought of `Pair`, that was kind of OK for this very specific case. In a future polishing pass we may want to rescue the `ProcessingPass` idea and make refactor all these functions to make use of it, for consistency.